core/src/main/java/org/bitcoinj/core/PeerSocketHandler.java

```diff
@@ -244,9 +244,9 @@ private void exceptionCaught(Exception e) {
         String s = addr == null ? "?" : addr.toString();
         if (e instanceof IOException) {
             // Short message for network errors
-            log.info(s + " - " + e.getMessage());
+            log.info("{} - {}", s, e.getMessage());
         } else {
-            log.warn(s + " - ", e);
```

Comment:
This won't work. It should actually generate another warning in your IDE because the number of arguments doesn't match the number of template variables.

Maybe we should change the line to use `e.getMessage()` just like the above?

Reply:
Ooops, typo. For some reason my IDE is not complaining. I'll fix it.

It seems Matt wanted the shorter message for network errors and the longer message for (less expected) errors. I think we should leave it that way (at least in this commit)

Reply:
The IDE was right, I think: https://slf4j.org/faq.html#paramException

Reply:
I changed the log message and force-pushed.